### PR TITLE
Add the -moz-border-*-colors properties

### PR DESCRIPTION
**Pull request by chrisdavidmills**
I've included a note as per https://bugzilla.mozilla.org/show_bug.cgi?id=1417200

### CODE REVIEW

File: css/properties/-moz-border-bottom-colors.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "1",
+              "version_removed": "59",
+              "notes": "In Firefox 59 the property was limited to usage in chrome code only."
```
Comment (teoli2003) [inline css/properties/-moz-border-bottom-colors.json]: "From Firefox 59, the property is..."

File: css/properties/-moz-border-left-colors.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "1",
+              "version_removed": "59",
+              "notes": "In Firefox 59 the property was limited to usage in chrome code only."
```
Comment (teoli2003) [inline css/properties/-moz-border-left-colors.json]: "From Firefox 59, the property is..."

File: css/properties/-moz-border-left-colors.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            },
+            "firefox_android": {
+              "version_added": "4",
+              "version_removed": "59",
+              "notes": "In Firefox 59 the property was limited to usage in chrome code only."
```
Comment (teoli2003) [inline css/properties/-moz-border-left-colors.json]: "From Firefox 59, the property is..."

File: css/properties/-moz-border-right-colors copy.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "1",
+              "version_removed": "59",
+              "notes": "In Firefox 59 the property was limited to usage in chrome code only."
```
Comment (teoli2003) [inline css/properties/-moz-border-right-colors copy.json]: "From Firefox 59, the property is..."

File: css/properties/-moz-border-right-colors copy.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+            },
+            "firefox_android": {
+              "version_added": "4",
+              "version_removed": "59",
+              "notes": "In Firefox 59 the property was limited to usage in chrome code only."
```
Comment (teoli2003) [inline css/properties/-moz-border-right-colors copy.json]: "From Firefox 59, the property is..."

File: css/properties/-moz-border-top-colors.json [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "1",
+              "version_removed": "59",
+              "notes": "In Firefox 59 the property was limited to usage in chrome code only."
```
Comment (teoli2003) [inline css/properties/-moz-border-top-colors.json]: "From Firefox 59, the property is..."

File: css/properties/-moz-border-top-colors.json [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+            },
+            "firefox_android": {
+              "version_added": "4",
+              "version_removed": "59",
+              "notes": "In Firefox 59 the property was limited to usage in chrome code only."
```
Comment (teoli2003) [inline css/properties/-moz-border-top-colors.json]: "From Firefox 59, the property is..."

**Comment by chrisdavidmills** [comment]
Done!

**Pull request merged**